Fix nightly 2025-06-11

Publish to ghcr.io 425 failed:

https://github.com/gardenlinux/gardenlinux/actions/runs/15577843595/job/43868471835

logs:

```
2025-06-11T07:23:02.8500421Z Current runner version: '2.325.0'
2025-06-11T07:23:02.8524544Z ##[group]Runner Image Provisioner
2025-06-11T07:23:02.8525347Z Hosted Compute Agent
2025-06-11T07:23:02.8525982Z Version: 20250529.345
2025-06-11T07:23:02.8526571Z Commit: 22237e57af968c281070ffa8ed912c1b754f9a23
2025-06-11T07:23:02.8527273Z Build Date: 2025-05-29T20:18:30Z
2025-06-11T07:23:02.8527818Z ##[endgroup]
2025-06-11T07:23:02.8528470Z ##[group]Operating System
2025-06-11T07:23:02.8529174Z Ubuntu
2025-06-11T07:23:02.8529650Z 24.04.2
2025-06-11T07:23:02.8530166Z LTS
2025-06-11T07:23:02.8530629Z ##[endgroup]
2025-06-11T07:23:02.8531107Z ##[group]Runner Image
2025-06-11T07:23:02.8531715Z Image: ubuntu-24.04
2025-06-11T07:23:02.8532263Z Version: 20250602.3.0
2025-06-11T07:23:02.8533233Z Included Software: https://github.com/actions/runner-images/blob/ubuntu24/20250602.3/images/ubuntu/Ubuntu2404-Readme.md
2025-06-11T07:23:02.8534839Z Image Release: https://github.com/actions/runner-images/releases/tag/ubuntu24%2F20250602.3
2025-06-11T07:23:02.8535848Z ##[endgroup]
2025-06-11T07:23:02.8536965Z ##[group]GITHUB_TOKEN Permissions
2025-06-11T07:23:02.8538753Z Actions: write
2025-06-11T07:23:02.8539552Z Metadata: read
2025-06-11T07:23:02.8540057Z Packages: write
2025-06-11T07:23:02.8540517Z ##[endgroup]
2025-06-11T07:23:02.8542635Z Secret source: Actions
2025-06-11T07:23:02.8543402Z Prepare workflow directory
2025-06-11T07:23:02.8932641Z Prepare all required actions
2025-06-11T07:23:02.8970113Z Getting action download info
2025-06-11T07:23:03.2546505Z ##[group]Download immutable action package 'actions/cache@v4'
2025-06-11T07:23:03.2547630Z Version: 4.2.3
2025-06-11T07:23:03.2548654Z Digest: sha256:c8a3bb963e1f1826d8fcc8d1354f0dd29d8ac1db1d4f6f20247055ae11b81ed9
2025-06-11T07:23:03.2550197Z Source commit SHA: 5a3ec84eff668545956fd18022155c47e93e2684
2025-06-11T07:23:03.2550971Z ##[endgroup]
2025-06-11T07:23:03.4120460Z Download action repository 'gardenlinux/python-gardenlinux-lib@269fcc4e7c7cd2e1995b02c1646673d030e496e6' (SHA:269fcc4e7c7cd2e1995b02c1646673d030e496e6)
2025-06-11T07:23:03.6153164Z Getting action download info
2025-06-11T07:23:03.7643446Z ##[group]Download immutable action package 'actions/setup-python@v5'
2025-06-11T07:23:03.7644472Z Version: 5.6.0
2025-06-11T07:23:03.7645386Z Digest: sha256:0b35a0c11c97499e4e0576589036d450b9f5f9da74b7774225b3614b57324404
2025-06-11T07:23:03.7646636Z Source commit SHA: a26af69be951a213d495a4c3e4e4022e16d87065
2025-06-11T07:23:03.7647573Z ##[endgroup]
2025-06-11T07:23:03.9710194Z Complete job name: Publish container base image / Update OCI manifest index
2025-06-11T07:23:04.0424659Z ##[group]Run actions/cache/restore@v4
2025-06-11T07:23:04.0426086Z with:
2025-06-11T07:23:04.0426810Z   path: manifests
2025-06-11T07:23:04.0427648Z   key: oci-manifests-15577843595
2025-06-11T07:23:04.0428709Z   restore-keys: oci-manifests-15577843595-

2025-06-11T07:23:04.0429992Z   fail-on-cache-miss: true
2025-06-11T07:23:04.0430935Z   enableCrossOsArchive: false
2025-06-11T07:23:04.0431882Z   lookup-only: false
2025-06-11T07:23:04.0432959Z ##[endgroup]
2025-06-11T07:23:04.2555251Z Cache hit for: oci-manifests-15577843595
2025-06-11T07:23:04.3318684Z Received 7585 of 7585 (100.0%), 0.5 MBs/sec
2025-06-11T07:23:04.3320991Z Cache Size: ~0 MB (7585 B)
2025-06-11T07:23:04.3362361Z [command]/usr/bin/tar -xf /home/runner/work/_temp/e8c9709c-7064-416a-af9b-43fd83a91986/cache.tzst -P -C /home/runner/work/gardenlinux/gardenlinux --use-compress-program unzstd
2025-06-11T07:23:04.3429255Z Cache restored successfully
2025-06-11T07:23:04.3509288Z Cache restored from key: oci-manifests-15577843595-vmware-gardener_prod-arm64
2025-06-11T07:23:04.3739646Z ##[group]Run gardenlinux/python-gardenlinux-lib/.github/actions/setup@269fcc4e7c7cd2e1995b02c1646673d030e496e6
2025-06-11T07:23:04.3741652Z with:
2025-06-11T07:23:04.3742401Z   version: 0.7.0
2025-06-11T07:23:04.3743221Z ##[endgroup]
2025-06-11T07:23:04.3909481Z ##[group]Run actions/setup-python@v5
2025-06-11T07:23:04.3910512Z with:
2025-06-11T07:23:04.3911263Z   python-version: 3.13
2025-06-11T07:23:04.3912355Z   cache: pip
2025-06-11T07:23:04.3913128Z   check-latest: false
2025-06-11T07:23:04.3914207Z   token: ***
2025-06-11T07:23:04.3914991Z   update-environment: true
2025-06-11T07:23:04.3916157Z   allow-prereleases: false
2025-06-11T07:23:04.3917583Z   freethreaded: false
2025-06-11T07:23:04.3919143Z ##[endgroup]
2025-06-11T07:23:04.5585409Z ##[group]Installed versions
2025-06-11T07:23:04.5670108Z Successfully set up CPython (3.13.3)
2025-06-11T07:23:04.5672700Z ##[endgroup]
2025-06-11T07:23:04.6063074Z ##[error]No file in /home/runner/work/gardenlinux/gardenlinux matched to [**/requirements.txt or **/pyproject.toml], make sure you have checked out the target repository
2025-06-11T07:23:04.6267522Z Post job cleanup.
2025-06-11T07:23:04.6328804Z Cleaning up orphan processes
```

--- a/.github/workflows/build_flavor.yml
+++ b/.github/workflows/build_flavor.yml
@@ -78,7 +78,7 @@ jobs:
         run: make ${{ inputs.flavor }}-${{ inputs.arch }}-build
       - name: Determine CNAME
         id: cname
-        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@269fcc4e7c7cd2e1995b02c1646673d030e496e6 # pin@0.7.0
+        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@095332c92d0d3ed83be59db6f1d3e44f63fbc46d # pin@0.7.1
         with:
           flags: --cname ${{ inputs.flavor }}-${{ inputs.arch }} cname
       - name: Set CNAME

--- a/.github/workflows/build_flavors_matrix.yml
+++ b/.github/workflows/build_flavors_matrix.yml
@@ -26,7 +26,7 @@ jobs:
           submodules: true
       - id: matrix
         name: Generate flavors matrix
-        uses: gardenlinux/python-gardenlinux-lib/.github/actions/flavors_parse@269fcc4e7c7cd2e1995b02c1646673d030e496e6 # pin@0.7.0
+        uses: gardenlinux/python-gardenlinux-lib/.github/actions/flavors_parse@095332c92d0d3ed83be59db6f1d3e44f63fbc46d # pin@0.7.1
         with:
           flags: ${{ inputs.flags }}
           flavors_matrix: ${{ inputs.flavors_matrix }}

--- a/.github/workflows/build_kmodbuild_container.yml
+++ b/.github/workflows/build_kmodbuild_container.yml
@@ -31,7 +31,7 @@ jobs:
       - if: ${{ steps.build_container_cache.outputs.cache-hit == 'true' }}
         name: Determine CNAME
         id: cname
-        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@269fcc4e7c7cd2e1995b02c1646673d030e496e6 # pin@0.7.0
+        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@095332c92d0d3ed83be59db6f1d3e44f63fbc46d # pin@0.7.1
         with:
           flags: --cname container-${{ matrix.arch }} cname
       - if: ${{ steps.build_container_cache.outputs.cache-hit == 'true' }}

--- a/.github/workflows/download_flavors_images.yml
+++ b/.github/workflows/download_flavors_images.yml
@@ -46,7 +46,7 @@ jobs:
           echo "${{ inputs.version }}" | tee VERSION
       - name: Determine CNAME
         id: cname
-        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@269fcc4e7c7cd2e1995b02c1646673d030e496e6 # pin@0.7.0
+        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@095332c92d0d3ed83be59db6f1d3e44f63fbc46d # pin@0.7.1
         with:
           flags: --cname ${{ matrix.flavor }}-${{ matrix.arch }} cname
       - name: Set CNAME

--- a/.github/workflows/download_workflow_data.yml
+++ b/.github/workflows/download_workflow_data.yml
@@ -22,7 +22,7 @@ on:
         value: ${{ jobs.workflow_data.outputs.target }}
 jobs:
   workflow_data:
-    name: Download workflow JSON data from trigger
+    name: Download workflow JSON data from trigger (${{ inputs.run_id }})
     runs-on: ubuntu-24.04
     outputs:
       commit_id: ${{ steps.data.outputs.commit_id }}

--- a/.github/workflows/publish_oci_containers.yml
+++ b/.github/workflows/publish_oci_containers.yml
@@ -82,12 +82,12 @@ jobs:
           echo "${{ inputs.version }}" | tee VERSION
       - name: Determine CNAME (amd64)
         id: cname_amd64
-        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@269fcc4e7c7cd2e1995b02c1646673d030e496e6 # pin@0.7.0
+        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@095332c92d0d3ed83be59db6f1d3e44f63fbc46d # pin@0.7.1
         with:
           flags: --cname container-amd64 cname
       - name: Determine CNAME (ard64)
         id: cname_arm64
-        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@269fcc4e7c7cd2e1995b02c1646673d030e496e6 # pin@0.7.0
+        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@095332c92d0d3ed83be59db6f1d3e44f63fbc46d # pin@0.7.1
         with:
           flags: --cname container-arm64 cname
       - name: Set CNAMEs
@@ -250,7 +250,7 @@ jobs:
           echo "${{ inputs.version }}" | tee VERSION
       - name: Determine CNAME
         id: cname
-        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@269fcc4e7c7cd2e1995b02c1646673d030e496e6 # pin@0.7.0
+        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@095332c92d0d3ed83be59db6f1d3e44f63fbc46d # pin@0.7.1
         with:
           flags: --cname ${{ matrix.flavor }}-${{ matrix.arch }} cname
       - name: Set CNAME
@@ -266,7 +266,7 @@ jobs:
           mkdir "$CNAME"
           tar -C "$CNAME" -xzf "$CNAME.tar.gz"
       - name: Install python-gardenlinux-lib
-        uses: gardenlinux/python-gardenlinux-lib/.github/actions/setup@269fcc4e7c7cd2e1995b02c1646673d030e496e6 # pin@0.7.0
+        uses: gardenlinux/python-gardenlinux-lib/.github/actions/setup@095332c92d0d3ed83be59db6f1d3e44f63fbc46d # pin@0.7.1
       - name: Install cosign
         uses: sigstore/cosign-installer@v3.8.2
         with:
@@ -317,7 +317,7 @@ jobs:
           echo "${{ inputs.version }}" | tee VERSION
       - name: Determine CNAME
         id: cname
-        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@269fcc4e7c7cd2e1995b02c1646673d030e496e6 # pin@0.7.0
+        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@095332c92d0d3ed83be59db6f1d3e44f63fbc46d # pin@0.7.1
         with:
           flags: --cname ${{ matrix.flavor }}-${{ matrix.arch }} cname
       - name: Set CNAME
@@ -359,7 +359,7 @@ jobs:
             oci-manifests-${{ github.run_id }}-
           fail-on-cache-miss: true
       - name: Install python-gardenlinux-lib
-        uses: gardenlinux/python-gardenlinux-lib/.github/actions/setup@269fcc4e7c7cd2e1995b02c1646673d030e496e6 # pin@0.7.0
+        uses: gardenlinux/python-gardenlinux-lib/.github/actions/setup@095332c92d0d3ed83be59db6f1d3e44f63fbc46d # pin@0.7.1
       - name: Update index using glcli tool
         run: |
           export GL_CLI_REGISTRY_TOKEN=${{ secrets.GITHUB_TOKEN }}

--- a/.github/workflows/publish_s3.yml
+++ b/.github/workflows/publish_s3.yml
@@ -117,7 +117,7 @@ jobs:
           echo "${{ needs.workflow_data.outputs.version }}" | tee VERSION
       - name: Determine CNAME
         id: cname
-        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@269fcc4e7c7cd2e1995b02c1646673d030e496e6 # pin@0.7.0
+        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@095332c92d0d3ed83be59db6f1d3e44f63fbc46d # pin@0.7.1
         with:
           flags: --cname ${{ matrix.flavor }}-${{ matrix.arch }} cname
       - name: Set CNAME

--- a/.github/workflows/test_flavor_chrooted.yml
+++ b/.github/workflows/test_flavor_chrooted.yml
@@ -35,7 +35,7 @@ jobs:
           fail-on-cache-miss: true
       - name: Determine CNAME
         id: cname
-        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@269fcc4e7c7cd2e1995b02c1646673d030e496e6 # pin@0.7.0
+        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@095332c92d0d3ed83be59db6f1d3e44f63fbc46d # pin@0.7.1
         with:
           flags: --cname ${{ inputs.flavor }}-${{ inputs.arch }} cname
       - name: Set CNAME

--- a/.github/workflows/test_platform_flavor.yml
+++ b/.github/workflows/test_platform_flavor.yml
@@ -83,7 +83,7 @@ jobs:
           fail-on-cache-miss: true
       - name: Determine CNAME
         id: cname
-        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@269fcc4e7c7cd2e1995b02c1646673d030e496e6 # pin@0.7.0
+        uses: gardenlinux/python-gardenlinux-lib/.github/actions/features_parse@095332c92d0d3ed83be59db6f1d3e44f63fbc46d # pin@0.7.1
         with:
           flags: --cname ${{ inputs.flavor }}-${{ inputs.arch }} cname
       - name: Set CNAME

